cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/batch/sink/DBSink.java

```diff
@@ -91,6 +92,13 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
                                                                                 getJDBCPluginId(),
                                                                                 PluginProperties.builder().build());
     Preconditions.checkArgument(jdbcDriverClass != null, "JDBC Driver class must be found.");
+    try {
+      Preconditions.checkArgument(tableExists(jdbcDriverClass), "Invalid table name %s", dbSinkConfig.tableName);
+    } catch (Exception e) {
+      throw new IllegalArgumentException(e);
+    } finally {
+      DBUtils.cleanup(jdbcDriverClass);
```

Comment:
Unrelated to this PR - can you move `DBUtils.cleanup(driverClass);` in `destroy()` method also into `finally` block?